db/workflow_test.go

```diff
@@ -0,0 +1,232 @@
+// Package db - workflow tests
+// The following tests validate database functionality, the instance payload.
+// A postgres instance with the tink database schema is required in order to run
+// the following tests.
+package db
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+)
+
+// TestCreateWorkflow : validates the creation of a new workflow.
+func TestCreateWorkflow(t *testing.T) {
```

Comment:
ditto to rename setupWorkflowTests or something :D

Reply:
Done